Casks/vcmi.rb

```diff
@@ -11,7 +11,7 @@
   desc "Open-source engine for Heroes of Might & Magic III"
   homepage "https://vcmi.eu/"
 
-  depends_on macos: ">= :high_sierra"
+  depends_on macos: ">= :sonoma"
```

Comment:
since it's our own formula and not official Homebrew's, let's go with the earliest supported
```suggestion
  depends_on macos: ">= :catalina"
```

Reply:
Fair enough!  Done.